Dockerfile.tmpl

```diff
@@ -155,7 +155,7 @@ RUN pip install pycuda && \
 
 RUN pip install pysal && \
     pip install seaborn python-dateutil dask python-igraph && \
-    pip install pyyaml joblib husl geopy ml_metrics mne pyshp && \
+    pip install pyyaml joblib husl geopy mne pyshp && \
```

Comment:
Dropping `ml_metrics`. This is an [old package created by Ben Hamner](https://github.com/benhamner/Metrics). Last updated in 2015. Fails to install now. It has been failing to import for a long time. 

In our Python image currently released:

```
import ml_metrics
```

Fails with:

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/tmp/ipykernel_34/1657033094.py in <module>
----> 1 import ml_metrics

/opt/conda/lib/python3.7/site-packages/ml_metrics/__init__.py in <module>
----> 1 from quadratic_weighted_kappa import *
      2 from elementwise import *
      3 from auc import auc
      4 from average_precision import apk, mapk

ModuleNotFoundError: No module named 'quadratic_weighted_kappa'
```